src/invariant/AccountSubEntriesCountIsValidTests.cpp

```diff
@@ -0,0 +1,318 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "database/Database.h"
+#include "invariant/AccountSubEntriesCountIsValid.h"
+#include "invariant/InvariantDoesNotHold.h"
+#include "invariant/InvariantManager.h"
+#include "invariant/InvariantTestUtils.h"
+#include "ledger/LedgerTestUtils.h"
+#include "lib/catch.hpp"
+#include "main/Application.h"
+#include "test/TestUtils.h"
+#include "test/test.h"
+#include <random>
+#include <xdrpp/autocheck.h>
+
+using namespace stellar;
+using namespace stellar::InvariantTestUtils;
+
+static LedgerEntry
+generateRandomAccountWithNoSubEntries(uint32_t ledgerSeq)
```

Comment:
same thing than earlier - I think this is overkill : we have a better chance of reaching high code coverage manually